e2e/client/playwright/page-object-models/monitoring.ts

```diff
@@ -37,9 +37,10 @@ export class Monitoring {
             .click();
     }
 
-    async createArticleFromTemplate(template: string): Promise<void> {
+    async createArticleFromTemplate(template: string, slugline: string): Promise<void> {
```

Comment:
convert function signature to the following:
`async createArticleFromTemplate(template: string, options?: {slugline?: string}): Promise<void>`

First argument makes sense from the funciton name, second is not easy to guess. After the change it would be easy to see what it is since the call would be 

`await monitoring.createArticleFromTemplate('story', {slugline: 'new article'});`

Also make slugline optional so the helper still works if content profile doesn't have a slugline.